article/app/model/AutomaticFilters.scala

```diff
@@ -0,0 +1,26 @@
+package model
+
+import common.GuLogging
+import model.TopMentionEntity.TopMentionEntity
+
+object AutomaticFilters extends GuLogging {
+
+  def getAutomaticFilter(filter: Option[String]): Option[(TopMentionEntity, String)] = {
+    filter.flatMap { f =>
+      val filterEntity = f.split(":")
+      if (filterEntity.length == 2) {
+        val entityType = TopMentionEntity.withNameOpt(filterEntity(0).toUpperCase)
+        if (entityType.isEmpty) {
+          log.error(s"automaticFilter query parameter entity ${filterEntity(0)} is invalid")
+          None
+        } else {
+          log.debug(s"valid automaticFilter query parameter - ${f}")
+          Some(entityType.get, filterEntity(1))
+        }
+      } else {
+        log.error(s"automaticFilter query parameter is invalid for ${f}, the format is <type>:<name>")
```

Comment:
Not a blocking comment - interesting that we log things like this as an error, its not the first time we've done this in frontend, but to me it seems like this is caused by a bad request and not because of an issue in frontend so its weird to me to be logging this as an error instead of as info.

Reply:
You're absolutely right. How about changing it to be a `warn`?